app_dart/lib/src/foundation/utils.dart

```diff
@@ -24,6 +24,9 @@ Duration twoSecondLinearBackoff(int attempt) {
   return const Duration(seconds: 2) * (attempt + 1);
 }
 
+/// Name of the default git branch.
+const String kDefaultBranchName = 'master';
```

Comment:
nit: maybe this belongs in config rather than here

Reply:
what do you know, it already is: https://github.com/flutter/cocoon/blob/master/app_dart/lib/src/datastore/cocoon_config.dart#L167

Reply:
done.